conduct.md

```diff
@@ -62,12 +62,15 @@ Allegheny's honor code for all submitted work.
 
 * Acceptable Conduct (A):
   * Members shall
-    * Watch the course website and Slack workspace for changes to due dates.
```

Comment:
Can you please explain why this point was deleted

Reply:
I personally don't think it was redundant but I'm fine with your decision to remove it. Once merge conflicts are resolved, I will approve this PR